sdk/src/main/java/com/hedera/hashgraph/sdk/TokenUpdateTransaction.java

```diff
@@ -657,6 +684,7 @@ void initFromTransactionBody() {
         if (body.hasMetadata()) {
             tokenMetadata = body.getMetadata().getValue().toByteArray();
         }
```

Comment:
`builder.setKeyVerificationMode(tokenKeyVerificationMode.toProtobuf())` is missing in the `build` method.
Not sure it would work without it.

Reply:
Thanks @0xivanov 🙏🏻